servers/cromwell/jobs/controllers/jobs_controller.py

```diff
@@ -246,6 +245,13 @@ def format_scattered_task(task_name, task_metadata):
                     shard_index=shard.get('shardIndex'),
                     execution_events=_get_execution_events(
                         shard.get('executionEvents'))))
+            if min_start > _parse_datetime(shard.get('start')):
+                min_start = _parse_datetime(shard.get('start'))
+            if shard.get('executionStatus') not in ['Failed', 'Done']:
```

Comment:
[ToL] String literals used as constants in correctness-critical places makes me a bit uneasy... would it make sense to centralize these in an enum?